pcsx2/ImGui/FullscreenUI.h

```diff
@@ -15,6 +15,8 @@ struct Pcsx2Config;
 
 namespace FullscreenUI
 {
+	class HddCreateInProgress;
```

Comment:
Forward declaration isn't necessary. HddCreateInProgress is only used in FullscreenUI.cpp, where it is defined.